Cargo.toml

```diff
@@ -16,25 +16,20 @@ categories = ["game-development"]
 
 [dependencies]
 approx = "0.5"
-bevy_polyline = "0.9"
+bevy_polyline = "0.10"
 parry3d = "0.13"
-bevy = { version = "0.13", default-features = false, features = [
+bevy = { version = "0.14", default-features = false, features = [
     "bevy_asset",
     "bevy_pbr",
 ] }
-bevy_mod_picking = { version = "0.19", default_features = false }
-nalgebra = { version = "0.32", features = ["glam025"] }
+bevy_mod_picking = { version = "0.20", default-features = false }
+nalgebra = { version = "0.32", features = ["glam027"] }
 
 [dev-dependencies]
-bevy = { version = "0.13", default-features = false, features = [
+bevy = { version = "0.14", default-features = false, features = [
     "bevy_asset",
     "bevy_pbr",
     "bevy_winit",
     "x11",
     "tonemapping_luts",
 ] }
-
-[[example]]
```

Comment:
Why delete?

Reply:
It's not necessary and for some reason cargo was compiling things incorrectly because of it and removing it fixed it